components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultPostAuthenticationHandler.java

```diff
@@ -148,7 +148,7 @@ private void handlePostAuthenticationForMissingClaimsRequest(HttpServletRequest
                         missingClaims);
                 uriBuilder.addParameter(FrameworkConstants.SESSION_DATA_KEY,
                         context.getContextIdentifier());
-                uriBuilder.addParameter("spName",
+                uriBuilder.addParameter(FrameworkConstants.REQUEST_PARAM_SP,
```

Comment:
Are we sure that this 'spName' parameter is not being read anywhere else and this ensures backward compatibility ?

Reply:
Changed the components using spName accordingly.